packages/apollos-ui-prayer/src/screens/PrayerScreen/index.js

```diff
@@ -73,6 +73,14 @@ const PrayerScreen = ({
 
   const hasPrayed = data?.prayed?.success || prayer.isPrayed;
 
+  const requestorName = [
+    prayer.requestor?.nickName,
+    prayer.requestor?.firstName,
+    'request',
+  ].filter((name) => {
+    return name !== null && name !== 'null' && name !== undefined;
```

Comment:
```suggestion
    return name != null && name !== 'null'
```

`!= null` will catch both `null` and `undefined` :D

Reply:
Oh nice, good to know!